Assets/Scripts/Models/InputOutput/CameraData.cs

```diff
@@ -15,14 +14,14 @@ public struct Preset
 {
     public Vector3 position;
     public float zoomLevel;
-    public int zLevel;
+    public int currentLayer;
 }
 
 public class CameraData
 {
     public Vector3 position;
     public float zoomLevel;
-    public int zLevel;
+    public int currentLayer;
```

Comment:
Does anyone have better potential names for this variable? I think "Layer" is a bit to similar to "overlay", but currentLevel isn't great either?
maybe depthLayer or depthLevel?